### ISSUE TITLE
Peer review from Ellen

### ISSUE THREAD

**Issue by ellenhsieh** [css-research/fp-dailing616#5]
Thank you for your presentation on this interesting topic about how “mere reminder” effect of calorie labeling influences consumers’ behavior. The outlay of the poster is really neat and in order. There are not too many texts so that everything is not to crowded, which is easy to read. 

As for the sections in the posters, maybe figure 1, 2, 3 section can be named more properly, instead of just using figure 1,2,3. For the background and method section, it is well illustrated the research hypothesis and the process of conducting the analysis. I think the sign 1 to 4 is really nice to illustrate different ways of calorie labeling. 

As for plots, all of them are really clear and well-labeled. Also, for figure 1, 2, 3, it is nice to also informed the mean of the changes so that audience can have a sense of how calorie labeling change the consumers behavior. The conclusion for each plots is clear. 

To sum up, this poster did a great job on presenting this interesting topic, which uses simple words and short paragraphs to demonstrate the work. Especially, the plots and conclusion provide some interesting insights about consumer behavior.
